fractal_tasks_core/tasks/projection.py

```diff
@@ -79,18 +88,23 @@ def projection(
         )
 
     # Compute the new shape and pixel size
-    dest_on_disk_shape = _compute_new_shape(orginal_image)
+    dest_on_disk_shape, z_axis_index = _compute_new_shape(orginal_image)
 
     dest_pixel_size = orginal_image.pixel_size
     dest_pixel_size.z = 1.0
     logger.info(f"New shape: {dest_on_disk_shape=}")
 
     # Create the new empty image
-    new_ngff_image = original_ngff_image.derive_new_image(
+    from devtools import debug
+    from pathlib import Path
+
+    debug(zarr_url)
```

Comment:
TODO: Remove debugs

Reply:
These actually will make the task fail on import on real deployments, because debug is a dev-only dependency (which is something we can't easily test for in the CI, because we rely on dev-only dependencies like pytest)

Reply:
I have added this to the pre-commits